edxval/serializers.py

```diff
@@ -100,7 +100,7 @@ def to_internal_value(self, data):
         Convert data into CourseVideo instance and image filename tuple.
         """
         course_id = data
-        course_video = image = None
+        course_video = image = ''
```

Comment:
The newest version of DRF (v3.6.3) is apparently stricter about empty strings vs. `None`. Defaulting to `None` and passing `None` through the DRF machinery caused a few tests to fail in edx-platform, while using an empty string fixed the tests.